<!DOCTYPE html>
<html lang='en'>


<head>
    <base href="..">
    <link rel="shortcut icon" type="image/png" href="assets/favicon.png"/>
    <link rel="stylesheet" type="text/css" media="all" href="assets/main.css"/>
    <meta name="description" content="Statistics Data Science Conference">
    <meta name="resource-type" content="document">
    <meta name="distribution" content="global">
    <meta name="KeyWords" content="Conference">
    <title>Program | Statistics Data Science Conference</title>
</head>


<body>

   <div class="banner">
        <img src="assets/banner.png" alt="Conference Template Banner" style="width: 98%">
        <div class="top-left">
<!--            <span class="title3">Statistics Data Science Conference </span> <!--<span class="title2">Template</span> <span class="year">2525</span> -->
        Statistics Data Science Conference 
        </div>
        <div class="bottom-right">
            June 02-05, 2023 <br> <font size="-2">UBC - Okanagan, Kelowna, BC </font>
        </div>
    </div>

    <table class="navigation">
        <tr>
            <td class="navigation">
                <a title="Conference Home Page" href=".">Home</a>
            </td>
            <td class="navigation">
                <a title="Register for the Conference" href="registration">Registration</a>
            </td>
            <td class="navigation">
                <a class="current" title="Conference Program" href="program">Program</a> 
            </td>
            <td class="navigation">
                <a title="Directions to the Conference" href="directions">Directions</a>
            </td>
            <td class="navigation">
                <a title="Conference Announcements" href="flyer">Announcements</a>
            </td>
        </tr>
    </table>

<!---
    <h2>Tentative Speakers (Ordered by Last Name)</h2>

    <style>
      ul {
        list-style-type: square;
        color:#999999;
      }
      ul li:before {
        content: '\2014';
        position: absolute;
        margin-left: -20px;
      }
    </style>

    <ul>
    	<li>Jiguo Cao - Simon Fraser University - Department of Statistics and Actuarial Science</li>
    	<li>Cong Chen - Merck & Co. Inc. (USA)</li>
    	<li>Xuanjin Cheng - Canada's Michael Smith Genome Sciences Cenlie (GSC) at BC Cancer</li>
    	<li>Gabriela V. Cohen Freue - University of British Columbia - Department of Statistics</li>
    	<li>Ehsan Karim - University of British Columbia - School of Population and Public Health</li>
    	<li>Mary Lesperance - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Longhai Li - University of Saskatchewan - Department of Mathematics and Statistics</li>
    	<li>Yang Liu - Veracyte, Inc.</li>
    	<li>Elham Majd - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Michelle Miranda - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Farouk Nathoo - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Ibrahim Numanagic - University of Victoria - Department of Engineering and Computer Science</li>
    	<li>Haolun Shi - Simon Fraser University - Department of Statistics and Actuarial Science</li>
    	<li>Jabed Tomal - Thompson Riveres University - Department of Statistics</li>
    	<li>Liangliang Wang - Simon Fraser University - Department of Statistics and Actuarial Science</li>
    	<li>Lang Wu - University of British Columbia - Department of Statistics</li>
    	<li>Li Xing - University of Saskatchewan - Department of Mathematics and Statistics</li>
    	<li>Shentu Yue - Daiichi Sankyo, Inc. (USA)</li>
    	<li>Xuekui Zhang - University of Victoria - Department of Mathematics and Statistics</li>
    	<li>Julie Zhou - University of Victoria - Department of Mathematics and Statistics</li>
    </ul>
--->



    <h2>Conference Program (speakers are ordered by last name)</h2>

    <table>

        <tr>
            <td class="date" rowspan="2">
                June 02 - June 04
            </td>
            <td class="title-special">
                Registration 
            </td>
        </tr>
        <tr>
            <td class="abstract">
           </td>
        </tr>
    </table>

    <table id="JiguoCao">
        <tr>
            <td class="date" rowspan="3">
            7.15.2022   
            </td>
            <td class="title">
                Deep Learning with Functional Inputs 
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Jiguo Cao (Simon Fraser University - Department of Statistics and Actuarial Science)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                We present a methodology for integrating functional data into deep neural networks. The model is defined for scalar responses with multiple functional and scalar covariates. A by-product of the method is a set of dynamic functional weights that can be visualized during the optimization process. This visualization leads to greater interpretability of the relationship between the covariates and the response relative to conventional neural networks. The model is shown to perform well in a number of contexts including prediction of new data and recovery of the true underlying relationship between the functional covariate and scalar response; these results were confirmed through real data applications and simulation studies.
	    </td>
        </tr>
    </table>


    <table id="XuanjinCheng">
        <tr>
            <td class="date" rowspan="3">
                
            </td>
            <td class="title">
                cSurvival: a web resource for biomarker interactions in cancer outcomes and in cell lines
            </td>
        </tr>
        <tr>
            <td class="speaker"  style="font-size: 19px">
                Xuanjin Cheng (Canada's Michael Smith Genome Sciences Centre [GSC] at BC Cancer)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Survival analysis is a technique for identifying prognostic biomarkers and genetic vulnerabilities in cancer studies. Large-scale consortium-based projects have profiled >11 000 adult and >4000 pediatric tumor cases with clinical outcomes and multiomics approaches. This provides a resource for investigating molecular-level cancer etiologies using clinical correlations. Although cancers often arise from multiple genetic vulnerabilities and have deregulated gene sets (GSs), existing survival analysis protocols can report only on individual genes. Additionally, there is no systematic method to connect clinical outcomes with experimental (cell line) data. To address these gaps, we developed cSurvival (https://tau.cmmt.ubc.ca/cSurvival). cSurvival provides a user-adjustable analytical pipeline with a curated, integrated database and offers three main advances: (i) joint analysis with two genomic predictors to identify interacting biomarkers, including new algorithms to identify optimal cutoffs for two continuous predictors; (ii) survival analysis not only at the gene, but also the GS level; and (iii) integration of clinical and experimental cell line studies to generate synergistic biological insights. To demonstrate these advances, we report three case studies. We confirmed findings of autophagy-dependent survival in colorectal cancers and of synergistic negative effects between high expression of SLC7A11 and SLC2A1 on outcomes in several cancers. We further used cSurvival to identify high expression of the Nrf2-antioxidant response element pathway as a main indicator for lung cancer prognosis and for cellular resistance to oxidative stress-inducing drugs. Altogether, these analyses demonstrate cSurvivalâ€™s ability to support biomarker prognosis and interaction analysis via gene- and GS-level approaches and to integrate clinical and experimental biomedical studies.
            </td>
        </tr>
    </table>

 
 
    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                
            </td>
            <td class="title">
                TBD
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Lucy Gao (University of British Columbia - Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                TBD
            </td>
        </tr>
    </table>
<!---
    <table>
        <tr>
            <td class="date" rowspan="2">
                2:50pm
            </td>
            <td class="title-special">
                Coffee &amp; Tea Break
            </td>
        </tr>
        <tr>
            <td class="abstract">
                <!-- Kept for Spacing 
            </td>
        </tr>
    </table>
-->
    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
            August 17    
            </td>
            <td class="title">
                TBD
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Juxin Liu (University of Saskatchewan - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                TBD
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                
            </td>
            <td class="title">
                TBD
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Xiaolin Huang (Amazon)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                TBD
            </td>
        </tr>
    </table>


    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                August 17
            </td>
            <td class="title">
                Ensembling Classification Models Based on Phalanxes of Variables with Applications in Drug Discovery
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Jabed Tomal (Thompson Riveres University - Department of Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Statistical detection of a rare class of objects in a two-class classification problem can pose several challenges. Because the class of interest is rare in the training data, there is relatively little information in the known class response labels for model building. At the same time the available explanatory variables are often moderately high dimensional. In the four assays of our drug-discovery application, compounds are active or not against a specific biological target, such as lung cancer tumor cells, and active compounds are rare. Several sets of chemical descriptor variables from computational chemistry are available to classify the active versus inactive class; each can have up to thousands of variables characterizing molecular structure of the compounds. The statistical challenge is to make use of the richness of the explanatory variables in the presence of scant response information. Our algorithm divides the explanatory variables into subsets adaptively and passes each subset to a base classifier. The various base classifiers are then ensembled to produce one model to rank new objects by their estimated probabilities of belonging to the rare class of interest. The essence of the algorithm is to choose the subsets such that variables in the same group work well together; we call such groups phalanxes.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
            7.15.2022    
            </td>
            <td class="title">
                Online Bayesian phylogenetic inference via sequential Monte Carlo 
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Liangliang Wang (Simon Fraser University - Department of Statistics and Actuarial Science)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Bayesian phylogenetics is to approximate a posterior distribution of phylogenetic trees based on biological sequence data such as DNA and RNA. Modern technologies can generate a large amount of sequence data every day. Most existing methods in Bayesian phylogenetics are costly to conduct inference for streaming data because they must start the computation anew each time when a new sequence is available. In this work, we propose an efficient online Bayesian phylogenetic method which can update an existing posterior with new sequences.  The proposed method is based on a sequential Monte Carlo method with a novel guided proposal distribution. 
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                
            </td>
            <td class="title">
                Robust Joint Modelling of Left-Censored Longitudinal Data and Survival Data, with Application to HIV Vaccine Studies
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Lang Wu (University of British Columbia - Department of Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                In jointly modelling longitudinal and survival data, the longitudinal data may be complex in the sense that they may contain outliers and may be left censored. Motivated from an HIV vaccine study, we propose a robust method for joint models of longitudinal and survival data, where the outliers in longitudinal data are addressed using a multivariate t-distribution for b-outliers and using an M-estimator for e-outliers. We also propose a computationally efficient method for approximate likelihood inference. The proposed method is evaluated by simulation studies. Based on the proposed models and method, we analyze the HIV vaccine data and find a strong association between longitudinal biomarkers and the risk of HIV infection.
            </td>
        </tr>
    </table>

    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                
            </td>
            <td class="title">
                A novel machine learning approach for gene module identification and prediction via a co-expression network of single-cell sequencing data
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Li Xing (University of Saskatchewan - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
                Gene co-expression network (GCN) analysis is widely used in microarray and RNA sequencing data to group genes with correlated expression levels, inferring similarity in function or coregulation in the pathway. In literature, such approaches are mainly unsupervised, which may introduce instability and variation across different datasets. Inspired by modern machine learning, we propose a novel approach that integrates supervised and unsupervised learning and simultaneously targets two tasks for analysis of RNA sequencing data: (1) gene module identification; and (2) phenotype prediction.<br>
		The identified gene modules from this approach could enable other researchers to conduct follow-up studies to figure out causal genes. And the novel approach also improves the accuracy of predicting phenotypes. Its algorithm incorporates parallel computation and other strategies to ensure researchers can handle large-scale single-cell data using personal computers. We showcased its use for single-cell auto-annotation.
            </td>
        </tr>
    </table>


    <table id="PUTSPEAKERNAMEHERE">
        <tr>
            <td class="date" rowspan="3">
                
            </td>
            <td class="title">
                EPPS: a Novel Ensemble Test to Improve the Power of Genomic Studies
            </td>
        </tr>
        <tr>
            <td class="speaker" style="font-size: 19px">
                Xuekui Zhang (University of Victoria - Department of Mathematics and Statistics)
            </td>
        </tr>
        <tr>
            <td class="abstract">
         The traditional SNP-wise test for Genome-Wide Association Studies (GWASs) often involves testing associations between a disease outcome and millions of single nucleotide polymorphisms (SNPs) and applies multiple testing adjustments afterwards to control the false positive rate. Due to the curse of dimensionality and the limited sample size, many GWASs face a lack of power issues based on such an approach. We proposed EPPS, a novel ensemble test procedure to increase the power of GWASs. Using the multiple random data splits approach, EPPS is an ensemble two-step test procedure inspired by the ensemble machine learning approach and a traditional pilot study in genomic experiments. EPPS provides `one' p-value for each SNP by integrating results of all data splits, enabling FDR control using any standard multiple testing adjustment approach. Furthermore, automatically selects values of its parameters to optimize study power with a pre-hoc power analysis.            </td>
        </tr>
    </table>

 

    <h2>Students'talks </h2>

    <style>
      ul {
        list-style-type: square;
        color:#999999;
      }
      ul li:before {
        content: '\2014';
        position: absolute;
        margin-left: -20px;
      }
    </style>

    <ul>
    	<li>Yushan Hu - University of Victoria, Title here </li>
    	<li>next student</li>
    </ul>


  <table>
    <tr>
            <td class="date" rowspan="2">
                10 am June 05
            </td>
            <td class="title-special">
                Discussion of planning the next conference 
            </td>
        </tr>
        <tr>
            <td class="abstract">
            </td>
        </tr>
    </table>


    <footer>
        &copy; Conference Organizers
        &nbsp;|&nbsp; Design by <a href="https://github.com/mikepierce">Mike Pierce</a>
    </footer>

</body>
</html>

